### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nasacrawford**
Adding the lessons to the README page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
-1. Ethos of Open Science <-- This module
-2. Tools and Resources
-3. Open Data
-4. Open Software
-5. Open Results
+[Objectives for Ethos of Open Science](lessons/learningobjectives.md) 
+
+## Lession for Open Science
+
+The full list of objectives and lessons can be found in teh [lessons directory](lessons/).   The Ethos of Open Science module has the following lessons:
```
Comment (yochannah) [inline README.md:11]:
```suggestion
The full list of objectives and lessons can be found in the [lessons directory](lessons/).   The Ethos of Open Science module has the following lessons: 
```

**Review by yochannah — APPROVED** [review]

**Comment by yochannah** [comment]
Thanks @nasacrawford ! It looks good to me - I've requested review from one more team member and then we'll merge it in!

File: README.md (hunk 1 above)
Comment (aosman12) [reply README.md:11]: Change “lession” for Open Science to “Lesson” for Open Science

**Review by aosman12 — CHANGES_REQUESTED** [review]
all good ! 
Commented to correct a minor spell check “lession” to “lesson”

**Comment by msundukova** [comment]
looks good to me.

File: README.md [hunk 2]
```diff
@@ -1,19 +1,22 @@
...
+By the end of this module, learners will be familiar with the definitions central to open science and have explored some concrete examples of the benefits of open science principles and community. The course will include best practices for building open science communities, increasing collaboration, and introducing open principles to project design, as well as an overview of open science norms. This module will also explore the historical impact of “closed” science, and how open science seeks to create a more diverse and equitable scientific community.
 
-1. Ethos of Open Science <-- This module
-2. Tools and Resources
-3. Open Data
-4. Open Software
-5. Open Results
+[Objectives for Ethos of Open Science](lessons/learningobjectives.md) 
+
+## Lession for Open Science
```
Comment (yochannah) [inline README.md:9]:
```suggestion
## Lesson for Open Science
```

**Pull request merged**